lib/backburner/host-platform.ts

```diff
@@ -0,0 +1,53 @@
+export interface IPlatform {
+  setTimeout(fn: () => void, ms: number): any;
+  clearTimeout(id: any): void;
+  next(): any;
+  clearNext(timerId: any): void;
+  now(): number;
+}
+
+const SET_TIMEOUT = setTimeout;
+const NOOP = () => {};
+
+export function buildDefaultPlatform(flush: () => void): IPlatform {
```

Comment:
should we just configure `asap`?

Reply:
I’m concerned with swapping to asap as it adds another layer of queuing and the exact ordering of interleaved RSVP promises, native promises, and BB queue transitions are super important...

However, this “host” interface style was explicitly designed to allow Ember to use RSVP.asap if it wants (it only needs to pass a `_buildPlatform` function that uses `RSVP.asap` for the provided `next` (you can see some examples in the tests)...